packages/scaffold-config/test/unit/detect.spec.ts

```diff
@@ -76,7 +76,7 @@ describe('detectFramework', () => {
     const projectPath = await scaffoldMigrationProject('create-react-app-unconfigured')
 
     fakeDepsInNodeModules(projectPath, [{ dependency: 'react-scripts', version: '5.0.0' }])
-    const actual = detectFramework(projectPath)
+    const actual = await detectFramework(projectPath)
```

Comment:
All of the test in this file were failing, the `detectFramework` was changed to be async recently but since we were never running on CI we didn't catch this

Reply:
Argh